tests/mellowLpRouter.test.ts

```diff
@@ -67,8 +68,17 @@ describe('Mellow Router Test Suite', () => {
     it('Check that vault has been initialised correctly', async () => {
       expect(ethMellowLpRouter.vaultInitialized).to.be.eq(true);
 
-      const tokenAddress = await localMellowRouterContract.token();
-      expect(ethMellowLpRouter.readOnlyContracts?.token.address).to.be.eq(tokenAddress);
+      // eslint-disable-next-line
+      console.log('printing chainid: ', network.config.chainId);
+      if (network.config.chainId === 1) {
```

Comment:
These tests are specific for Goerli so let's keep this check also Goerli-specific.

Reply:
Done